examples/campaign_management/add_complete_campaigns_using_mutate_job.py

```diff
@@ -0,0 +1,536 @@
+    text_ad.headline_part1.value = f'Cruise to Mars #{uuid4()}'
+    text_ad.headline_part2.value = 'Best Space Cruise Line'
+    text_ad.description.value = 'Buy your tickets now!'
+    final_url = ad_group_ad.ad.final_urls.add()
+    final_url.value = 'http://www.example.com'
+    ad_group_ad.ad_group.value = ad_group_operation.create.resource_name
+    ad_group_ad.status = client.get_type('AdGroupAdStatusEnum',
+                                         version='v3').PAUSED
+
+    return ad_group_ad_operation
+
+
+def run_mutate_job(mutate_job_service, resource_name):
+    """Runs the mutate job for executing all uploaded mutate job operations.
+
+    Args:
+        mutate_job_service: an instance of the MutateJobService message class.
+        resource_name: a str of a resource name for a mutate job.
+
+    Returns: a google.api_core.operation.Operation instance.
+    """
+    try:
+      response = mutate_job_service.run_mutate_job(resource_name)
+      print(f'Mutate job with resource name "{resource_name}" has been '
+            ' executed.')
+      return response
+    except GoogleAdsException as exception:
+      handle_google_ads_exception(exception)
+
+
+def poll_mutate_job(operations_response):
+    """Polls the server until the mutate job execution finishes.
+
+    Sets the initial poll delay time and the total time to wait before time-out.
+
+    Args:
+        operations_response: a google.api_core.operation.Operation instance.
+    """
+    # operations_response represents a Long-Running Operation or LRO. The class
+    # provides an interface for polling the API to check when the operation is
+    # complete. Below we use the synchronous interface, but there's also an
+    # asynchronous interface that uses the Operation.add_done_callback method.
+    # See: https://googleapis.dev/python/google-api-core/latest/operation.html
+    operations_response.result(timeout=MAX_TOTAL_POLL_INTERVAL_SECONDS)
+
+
+def fetch_and_print_results(mutate_job_service, resource_name):
+    """Prints all the results from running the mutate job.
+
+    Args:
+        mutate_job_service: an instance of the MutateJobService message class.
+        resource_name: a str of a resource name for a mutate job.
+    """
+    print(f'Mutate job with resource name "{resource_name}" has finished. '
+          'Now, printing its results...')
+
+    # Gets all the results from running mutate job and prints their information.
+    mutate_job_results = mutate_job_service.list_mutate_job_results(
+        resource_name, page_size=PAGE_SIZE)
+
+    for mutate_job_result in mutate_job_results:
+        status = mutate_job_result.status.message
+        status = status if status else 'N/A'
+        result = mutate_job_result.mutate_operation_response
+        result = result if result.ByteSize() else 'N/A'
+        print(f'Mutate job #{mutate_job_result.operation_index} '
+              f'has a status {status} and response type {result}')
```

Comment:
I think quotes around the status and result values are useful.

Reply:
Done.